### DISCUSSION TITLE
Why use `space-` over `gap-`?

### DISCUSSION THREAD

**Discussion by nichtsam** [shadcn-ui/ui#2023] [General]
I see a lot there are a lot of usage of tailwind utility className `space-` in flexBoxes.
since it's already in a flexBox, I'm just curious why not use `gap-`?
In my opinion, it does the same thing, and makes it easier to understand the styling when checking through devTools.
is it about consistency?

**Comment by shrix1** [comment]
I feel like `gap` is probably the best choice when using flex or grid because, in some cases, `space` can disrupt the UI due to _its reliance on margins_.

**Reply by nichtsam** [reply]
yeah me too.
I've change all the `space` into `gap` in my ui code, it feels more intuitive to me this way.
